frontend/src/app/pages/lobby-page/lobby-page.component.ts

```diff
@@ -217,17 +217,22 @@ export class LobbyPageComponent implements OnInit, OnDestroy {
     // Function to return all other users in the room
     private getOtherUsersInLobby() {
 
+      console.log("first trigger");
```

Comment:
Was this finished? Might want to remove the debugging logs if so :)

Reply:
It wasn't finished. Was trying to play with it and figure it out, but no dice.

Reply:
Ah forsure, no worries! Let's figure it out monday, and then pull this in 👍